src/firebaseConfig.js

```diff
@@ -1,5 +1,6 @@
-import * as firebase from "firebase";
-import "firebase/database";
+import firebase from "firebase/app";
+import "@firebase/auth";
```

Comment:
i dont think line 2 and 3 are needed

Reply:
if i delete them, the register and login are not working

Reply:
deleted